File: SharedApi/Models/Rank.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SharedApi.Models
{
	public partial class Rank
	{
		public int ID { get; set; }
		public string Title { get; set; }
		public int FromXP { get; set; }
		public int? ToXP { get; set; }
		public string Colour { get; set; }
	}
}

File: MobileApp/zoliky_teachers/lib/utils/api/models/Rank.dart
import 'package:zoliky_teachers/utils/interfaces/IDbObject.dart';

class Rank implements IDbObject {
  int id;
  String title;
  int fromXP;
  int toXP;
  String colour;

  Rank.fromJson(Map<String, dynamic> json)
      : id = json["ID"],
        title = json["Title"],
        fromXP = json["FromXP"],
        toXP = json["ToXP"],
        colour = json["Colour"];
}
